index.js

```diff
@@ -1,4 +1,7 @@
 /**
- * @exports async-domino
+ * @module transformer
  */
-module.exports = require('./lib/main');
+module.exports = {
+    lib: require('./lib/main'),
+    util: require('./lib/util')
+};
```

Comment:
Doesn't this change the entire API?

`lib/main` could export an additional `util` beside the expected `convert` and `validate`? This also makes index.js static (which is intended) so that API changes are done via main.